src/documentation.js

```diff
@@ -26,13 +26,70 @@ function _mapToObj(map) {
   return returnObj;
 }
 
+/**
+ * maps requests to delete to an array containing the promise and the body of the request
+ * @param {aws} provider
+ * @param {documentationPieces} Object, contains documentationToUpsert and documentationToDelete
```

Comment:
please update: remove documentationPieces and add documentationToUpsert and documentationToDelete

Reply:
read a couple things on JSDoc, I'll update comments